### ISSUE TITLE
Crashes OS when running application through terminal

### ISSUE THREAD

**Issue by kev-zheng** [Thomas101/wmail#655]
* Which version of WMail are you using? 2.3.0

* Which Operating System are you using? Fedora 25 with GNOME 3
Was testing WMail out by launching from command line. I closed the app but the application was still running in the terminal, so I closed it with ctrl+c, which crashed with a black screen

**Comment by Thomas101** [comment]
Hi did it bring down the whole OS? If you try it again do you see the same behaviour?

**Comment by kev-zheng** [comment]
Yes, it sends me to the login screen and all applications are closed on login

**Comment by Thomas101** [comment]
Okay, thanks. I'll fire up a fedora machine and see if I can reproduce
